core/frontend/src/tile/TileAdmin.ts

```diff
@@ -160,15 +162,16 @@ export class TileAdmin {
    * @param props Options for customizing the behavior of the TileAdmin.
    * @returns the TileAdmin
    */
-  public static create(props?: TileAdmin.Props): TileAdmin {
-    return this.createForDeviceType(ProcessDetector.isMobileBrowser ? "mobile" : "non-mobile", props);
+  public static async create(props?: TileAdmin.Props): Promise<TileAdmin> {
+    const rpcConcurrency = IpcApp.isValid ? (await IpcApp.callIpcHost("queryConcurrency", "cpu")) : undefined;

```

Comment:
unfortunately i don't think this will work for people who call `TileAdmin.create` before calling `IpcApp.startup` (see TileTree.test.ts) because `IpcApp.isValid` isn't true until you call startup.

Reply:
Then I'll change `IModelAppOptions.tileAdmin` to be a `TileAdmin.Props` instead of a `TileAdmin`.
Is it weird that people can call `IpcApp.startup` without supplying an `IpcSocketFrontend`?

Reply:
Made that change.

Reply:
> Is it weird that people can call IpcApp.startup without supplying an IpcSocketFrontend?

Yes, i don't know why that's allowed. I will address in my other PR.